src/schema.js

```diff
@@ -0,0 +1,99 @@
+export function buildSchema() {
```

Comment:
Ik denk dat hier iets in zat waardoor het een builder functie moest zijn, maar het lijkt erop dat je dat eruit gerefactored hebt. Kan volgensmij gewoon weer een export van een `const` zijn :)